package.json

```diff
@@ -32,7 +32,7 @@
   ],
   "dependencies": {
     "codemod-cli": "^2.1.0",
-    "ember-codemods-telemetry-helpers": "^1.0.0",
+    "ember-codemods-telemetry-helpers": "github:tylerturdenpants/ember-codemods-telemetry-helpers#fix/exit-too-early",
```

Comment:
This is the actual bugfix for #33.
The rest of this PR is just working on the test harness.

Reply:
https://github.com/ember-codemods/ember-codemods-telemetry-helpers/releases/tag/v1.0.1 is released

Reply:
yay!!!